src/XamlStyler.UnitTests/TestConfigurations/AllDifferent.json

```diff
@@ -27,5 +27,6 @@
   "ThicknessAttributes": "Margin",
   "FormatOnSave": false,
   "SaveAndCloseOnFormat": false,
-  "CommentPadding": 3
+  "CommentPadding": 3,
+  "IgnoreDesignTimeReferencePrefix": false
```

Comment:
Our documentation is not great in these test files -- for this one, we change each option to something _other_ than the default.

Reply:
Corrected

Reply:
@grochocki I considered and applied all your suggestions as well as added unit test for option I added :)